currency/l1/public.yml

```diff
@@ -159,3 +159,102 @@ paths:
             text/html:
               schema:
                 type: string
+
+  /validate:
+    post:
+      tags:
+        - Currency
+      description: Validate signed data update.
+      requestBody:
+        description: Signed data update.
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '../../models/schemas/SignedDataUpdate.yml'
+      responses:
+        204:
```

Comment:
I would suggest a 200 and an empty JSON response object for this endpoint. It would allow us to add response data in the future without having to alter the response code and change from no content to some content.

Reply:
200
{}

Reply:
/data/validate